script.js

```diff
@@ -1,4 +1,232 @@
+// var main = function (input) {
+//   var myOutputValue = 'hello world';
+//   return myOutputValue;
+// };
+
+//Create a basic version of Scissors Paper Stone where the user inputs one of "scissors", "paper", or "stone", the program internally randomly chooses scissors, paper, or stone, and the program outputs whether the user won, the program won, or it's a draw.
+
+//Basic Scissors Paper Stone Base
+var numOfUserWinning = 0;
+var numOfProgramWinnning = 0;
+var numOfDraws = 0;
+var isGameStarted = false;
+var userName = "";
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (!isGameStarted) {
+    return startGame(input);
+  }
+  return gameTime(input);
+};
```

Comment:
Amazing job keeping your main function super short with the use of helper functions! :)